src/Pact/Eval.hs

```diff
@@ -164,6 +161,17 @@ enforceGuard i g = case g of
       MDInterface{} -> evalError' i $ "ModuleGuard not allowed on interface: " <> pretty mg
   GUser UserGuard{..} ->
     void $ runSysOnly $ evalByName _ugFun _ugArgs (getInfo i)
+  GCapability CapabilityGuard{..} -> do
+    maybe (return ()) (enforcePactId True) _cgPactId
```

Comment:
```suggestion
    forM_ _cgPactId $ \pid ->
      enforcePactId True pid
```

Reply:
I didn't compile this locally, you may need to import `Data.Foldable`...

Reply:
I see you used `traverse_`, aka the same idiom with flipped args. Looks good to me!